kolibri/plugins/user_profile/assets/src/views/ProfilePage/index.vue

```diff
@@ -301,8 +301,8 @@
         message: 'Move your account and progress data to another learning facility.',
         context: 'Explanation of what change a learning facility means',
       },
-      learn: {
-        message: 'Learn',
+      learnMore: {
```

Comment:
This string is also used in the MissingResourceAlert component, but I don't think there's much value in consolidating this just yet.

Reply:
Ok, just 2 words, we can let it slip for now.